aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,58 @@
+        s3.delete_objects(
+            Bucket=bucket_name,
+            Delete={"Objects": [{"Key": obj["Key"]} for obj in objects["Contents"]]}
+        )
+    print(f"All objects deleted from {bucket_name}")
+except Exception as e:
+    print(f"Error deleting objects: {e}")
+
+try:
+    s3.delete_bucket(Bucket=bucket_name)
+    print(f"Bucket {bucket_name} deleted successfully.")
+except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)